transport_nantes/topicblog/views.py

```diff
@@ -376,9 +390,16 @@ class TopicBlogEmailView(TopicBlogBaseView):
     model = TopicBlogEmail
 
 
-class TopicBlogEmailViewOne(TopicBlogBaseViewOne):
+class TopicBlogEmailViewOne(TopicBlogEmailViewOnePermissions,
+                            TopicBlogBaseViewOne):
     model = TopicBlogEmail
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['context_appropriate_base_template'] = 'asso_tn/base_mobilitain.html'
```

Comment:
Why displaying emails with base_mobilitain ? It's quite different from the original template and the template the user is supposed to see in his client. I feel like viewing an email object should be seen as if if was in a mail client. Unless you have some motives I haven't seen yet ?

Reply:
The view is in the context that the user has clicked the "display in my browser", so they are now displaying as part of our website.  The should have a standard web experience.

The other rendering is when we send the content as email, then we use `base_email_mobilitain` to render for an email client.

Reply:
Ha, didn't  see base-email_mobilitain yet, this explains it ! I get it, SGTM

Reply:
That said, we can fiddle with it later to get it how we want.  Maybe we'll want reduced headers/menus for displaying emails.  I was envisioning something akin to how we display press releases or blog posts.